scripts/20230519_add_target_account_id/script.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+
+// BEFORE
+// Count total number of payments with no received webhooks. This should be 0.
+db.getCollection('payments').countDocuments({ webhookLog: { $size: 0 } }) === 0
+
+// Count total number of payments.
+const total = db.getCollection('payments').countDocuments({})
+
+// Count total number of payments with no targetAccountId value. This should be equal to the total number of payments.
+db.getCollection('payments').countDocuments({ targetAccountId: { $exists: false } }) === total
+
+// UPDATE
+db.getCollection('payments').update(
```

Comment:
```suggestion
db.getCollection('payments').updateMany(
```
Got this error when trying to run the script on my Studio3T:
```sh
DeprecationWarning: Collection.update() is deprecated. Use updateOne, updateMany, or bulkWrite.
```

Reply:
yes fixed! [6f05a50](https://github.com/opengovsg/FormSG/pull/6360/commits/6f05a50b60029d851d39951de780bb02a25fba4d)